packages/client/src/components/preview/SelectionIndicator.svelte

```diff
@@ -5,10 +5,14 @@
   $: color = $builderStore.editMode
     ? "var(--spectrum-global-color-static-green-500)"
     : "var(--spectrum-global-color-static-blue-600)"
+
+  $: isScreen = $builderStore.selectedComponentId?.startsWith(
+    $builderStore.screen._id
+  )
```

Comment:
We don't need these changes, since there is no longer any matching DOM nodes for the screen ID, so we can always just let the indicator render with whatever the selected component ID is. Think I said that in my initial review but must have just forgotten to remove it 👍

Reply:
Apologies, I completely forgot to add this to the commit.